app/templates/components/user-avatar.hbs

```diff
@@ -1,9 +1,9 @@
- <div class="profile">
-    {{#if user.image}}
-      <img src={{user.image}} alt="user">
-    {{else}}
-      <div class="profile-icon" style={{isClosing}}>
-        <p>{{profile_char}}</p>
-      </div>
-    {{/if}}
-  </div>
\ No newline at end of file
+<div class="profile">
+  {{#if user.image}}
+  <img src={{user.image}} alt="user">
+  {{else}}
+  <div class="profile-icon" style={{isClosing}}>
```

Comment:
Can you explain what this isClosing does ..?

Reply:
isClosing is a computed property for dynamic background style and Now I have gave the proper name for that computed property.